web_calendar_custom/static/src/js/web_calendar_custom.js

```diff
@@ -0,0 +1,143 @@
+/*
+    © 2018 Savoir-faire Linux <https://savoirfairelinux.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+
+
+odoo.define('web_calendar_custom.renderer_setting', function (require) {
+    "use strict";
+
+    var rpc = require('web.rpc');
+    var AbstractRenderer = require('web.AbstractRenderer');
+    var core = require('web.core');
+
+    var _t = core._t;
+    var config_values = null;
+
+    /* Override fullCalendar configuration */
+    AbstractRenderer.include({
+
+        willStart: function () {
+
+            /* Get configuration values from res.config.settings */
+            return rpc.query({
+                model: 'res.config.settings',
+                method: 'get_values',
```

Comment:
Use this method can have an impact in the performance... please, refactor the python part to separate the logic that prepare the values.